### PR TITLE
macOS build

### PR DESCRIPTION
**Pull request by SeanAsby**

### CODE REVIEW

File: super_rust_boy_c_binding/src/lib.rs [hunk 1]
```diff
@@ -64,7 +64,7 @@ pub extern fn rustBoyCreate(cartridge_path: *const c_char, save_file_path: *cons
         }
     };
 
-    let instance = RustBoy::new(cart_path, save_path, UserPalette::Default, false);
+    let instance = RustBoy::new(cart_path, save_path, UserPalette::Default, true);
```
Comment (coopersimon) [inline super_rust_boy_c_binding/src/lib.rs]: Would be cool to not mute on macOS

**Review by coopersimon — CHANGES_REQUESTED** [review]

File: super_rust_boy_c_binding/src/lib.rs (hunk 1 above)
Comment (SeanAsby) [reply super_rust_boy_c_binding/src/lib.rs]: This change was already merged in the other PR.

Comment (coopersimon) [reply super_rust_boy_c_binding/src/lib.rs]: Would it be possible to take the mute parameter from the outside so on macos builds we can not mute?

Comment (SeanAsby) [reply super_rust_boy_c_binding/src/lib.rs]: Yes! I'll look into it

Comment (SeanAsby) [reply super_rust_boy_c_binding/src/lib.rs]: @coopersimon Exposed mute now

**Review by coopersimon — APPROVED** [review]

**Pull request merged**